packages/e2e-tests/specs/editor/blocks/__snapshots__/buttons.test.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`Buttons can jump to the link editor using the keyboard shortcut 1`] = `
 "<!-- wp:buttons -->
 <div class=\\"wp-block-buttons\\"><!-- wp:button -->
-<div class=\\"wp-block-button\\"><a class=\\"wp-block-button__link\\">WordPress</a></div>
+<div class=\\"wp-block-button\\"><a class=\\"wp-block-button__link\\" href=\\"https://wwww.wordpress.org/\\" title=\\"\\">WordPress</a></div>
```

Comment:
```suggestion
<div class=\\"wp-block-button\\"><a class=\\"wp-block-button__link\\" href=\\"https://www.wordpress.org/\\" title=\\"\\">WordPress</a></div>
```

Reply:
Hm, I swear I'd made this change before 🤔 But thanks for catching it and the suggestion.